helm-chart/sefaria-project/templates/rollout/nginx.yaml

```diff
@@ -96,6 +96,8 @@ spec:
             value: "varnish-{{ .Values.deployEnv }}-{{ .Release.Revision }}"
           - name: SEARCH_HOST
             value: "{{ .Values.nginx.SEARCH_HOST }}"
+          - name: STRAPI_LOCATION
+            value: "{{ .Values.localSettings.STRAPI_LOCATION }}"
```

Comment:
This needs 
1) For .Values.localSettings.STRAPI_LOCATION to be defined in the values.yaml file of the chart.  You can populate with a sensible default if all Sefaria environments will redirect to the same endpoint
2) per-environment values to populated in the environment values files - build/ci/production-values.yaml for prod, build/ci/sandbox-values.yaml for per-commit tests, build/ci/integration-values.yaml for integration tests and create-cauldron.sh in the cauldrons repo for cauldrons.